repo/packages/M/miniod/1/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "packagingVersion": "4.0", 
+    "upgradesFrom": [
+        "{{upgrades-from}}"
+    ], 
+    "downgradesTo": [
+        "{{downgrades-to}}"
+    ], 
+    "minDcosReleaseVersion": "1.10", 
+    "name": "miniod", 
+    "version": "0.1.1", 
```

Comment:
hi @Gupta-Amrit, version is missing the basetech version part, see previous release for sample.

cc @ryadav88

Reply:
Done.